usage/rest-api/src/main/java/org/apache/brooklyn/rest/domain/CatalogItemSummary.java

```diff
@@ -121,6 +132,20 @@ public String getIconUrl() {
         return iconUrl;
     }
 
+    public Collection<Object> getTags() {
+        List<Object> result = new ArrayList<Object>();
+        for (Object t : tags) {
+            // TODO if we had access to a mapper we could use it to give better json
```

Comment:
This method was taken [from there](https://github.com/apache/incubator-brooklyn/blob/master/usage/rest-api/src/main/java/org/apache/brooklyn/rest/domain/TaskSummary.java#L137-L144). Although, simply returning the tags seems to work as expected. I'll change it.